added in pink background/ red border for passwords less than 10 chars

shirlyfang opened the pull request:
Added in lightpink background color and red border for passwords < 10 chars. It seems like eslint made some formatting changes when git commit was ran. 
<img width="586" alt="mpsignupscreenshot" src="https://user-images.githubusercontent.com/29534862/48317853-150bd500-e5ad-11e8-81fd-feff067d62d2.png">

nkansal96 commented on src/components/Register/detailsCard.js:
```diff
@@ -1,53 +1,79 @@
+      const changed = e.target.name;
+      const changedLength = e.target.value.length;
+      if (changed === 'password') {
+        changedLength < 10 ? this.setState({
```
Put the ternary inside the object passed to setState to avoid this awkward syntax of calling a function with a different argument and using its side effect as a result

alexanderqchen replied:
I agree with this! It looks a little messy right now

nkansal96 commented on src/components/Register/style.scss:
```diff
@@ -1,245 +1,261 @@
+  }
+  .input-invalid {
+    width: 383px;
+    border: 1px solid #ff0000;
```
I think we use an existing variable for the red color that’s consistent across the UI. You should use that instead, or create a variable if it doesn’t exist.

alexanderqchen replied:
yup `src/styles/vars.scss` has a lot of styling variables to keep things consistent. Your probably looking for $light-red

alexanderqchen commented:
@nkansal96 I actually requested for the pink background color in addition to the red border because I initially thought just the border didn't make it clear enough. But after playing with it on my machine, I think red border AND making the "password" text red should be clear enough and stay consistent with the existing design. I'll include that in my review.

alexanderqchen commented:
Oh wait, it also seems like your changes aren't passing eslint. Did eslint not tell you some code styling changes to make? It should be run automatically when you make a commit.

shirlyfang commented:
I made the following changes :
Put the ternary inside. 


Made changes to detailCard so that the red text will show when password onBlur or if in textbox and was previously <10. Once password length is >= 10, text is normal. 

Removed all eslint issues with detailCard.
Removed all but 6 issues(destructuring props, jsx/js) with index. 

I tried my best to try to use & in styles.scss so that I wouldn't have to redo all the code in invalid-length but it was not working. Tried to follow same structure as compare-results in styles and detailCard. 
<img width="527" alt="mpsignupscreenshot2" src="https://user-images.githubusercontent.com/29534862/48323835-f1638180-e5e2-11e8-8574-5c179f31081e.png">

alexanderqchen commented:
Good updates! Small thing I noticed, is when the password field is focused, the "password" text is still red, but it would be best if it also became normal color again. 
![screen shot 2018-11-15 at 5 59 00 pm](https://user-images.githubusercontent.com/12878275/48593146-5af3d080-e900-11e8-8f16-06f42c70d488.png)

alexanderqchen commented on src/components/Register/style.scss:
```diff
@@ -1,245 +1,277 @@
+      }
+    }
+  }
+  .invalid {
```
it seems like line 230 - 243 are redundant

alexanderqchen commented on src/components/Register/detailsCard.jsx:
```diff
@@ -0,0 +1,117 @@
+    this.handleFocus = this.handleFocus.bind(this);
+    this.state = {
+      passwordLength: 1,
+      passwordExited: 1,
```
i'm not completely sure what passwordExited is. it seems like its being used as a boolean, but has a number value. Also not sure what that number value means either

alexanderqchen commented:
Looks good! Thanks for the quick changes
